### ISSUE TITLE
Repeatedly rearraging subscription page if page-back from video

### ISSUE THREAD

**Issue by kombucha7** [Vulpelo/hide-youtube-shorts#15]
As per title, with experiemental option rearrange enabled, page will be re-rearranged everytime you page-back from a video to subscription page. 

Refreshing clears the effects.

Flow:

Subscription page(Start) -> Click any video -> page-back (repeat from step 2 if needed)

![image](https://github.com/Vulpelo/hide-youtube-shorts/assets/29064557/daabce6c-c99d-40c1-8704-c2b66b62c5ff)
![image](https://github.com/Vulpelo/hide-youtube-shorts/assets/29064557/167ef905-aed4-49f8-b573-4f03eed75048)

**Comment by Vulpelo** [comment]
Thanks for reporting the issue. I will fix this issue as soon as the new version of the YouTube subscription page is available in my country. Otherwise It will be hard to test since, from what I checked, the new grid (also being used on home page) uses some YT scripts that also are trying to rearrange videos when navigating back to home page.

**Comment by Vulpelo** [comment]
I finally got an updated subscription page. From what I analyzed, the new grid system is reusing grid elements when they are hidden (have attribute 'hidden') and causes to hide non shorts videos and then displaying multiple videos in a row.

I have made an update so when using 'rearrange videos' option, it removes shorts from the grid instead of hiding them (I was hiding them so there was no need to reload the page to show them again after disabling 'hide shorts' option, but seems that will no longer be an option). 

The new version 1.3.1 was pushed to plugin store.
